environment.yml

```diff
@@ -1,6 +1,7 @@
 name: coiled
 channels:
   - conda-forge
+  - defaults
```

Comment:
Do we need to include `defaults` for a package that isn't available on `conda-forge`? If not, let's remove this to avoid obscure errors in the future due to mixing conda channels. I'm happy to actually submit the PR for this, just double checking if adding `defaults` is intentional